web/cypress/fixtures/selector.json

```
@@ -11,14 +11,19 @@
   "errorNotification:": ".ant-notification-notice-error",
 
   "pluginCard": ".ant-card",
+  "pluginCardBordered": ".ant-card-bordered",
   "pageContent": ".ant-pro-page-container",
 
+  "drawerContent": ".ant-drawer-content",
```

```suggestion
  "drawer": ".ant-drawer-content",
```
We already have that variable, just leave one of the two.